Sources/Hummingbird/Storage/MemoryPersistDriver.swift

```diff
@@ -18,9 +18,35 @@ import NIOCore
 
 /// In memory driver for persist system for storing persistent cross request key/value pairs
 public actor MemoryPersistDriver<C: Clock>: PersistDriver where C.Duration == Duration {
+    public struct Configuration: Sendable {
+        /// amount of time between each call to tidy
+        public var tidyFrequency: Duration
```

Comment:
Can we bundle the clock in this Configuration too? Since it's within the Generic context of `MemoryPersistDriver`, this type is generic too.

Reply:
Can't easily do that and have a default variable for the clock

Reply:
Ah, bummer.